docs/content/stepbystep/IDE-setup.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: 'IDE Setup'
+metaTitle: 'IDE Setup | Discordeno'
+metaDescription: 'Setup VSC for discordeno'
+---
+
+# Installing
+
+There are many editors for js/ts, but I recommend Visual Studio Code as its free and fast with a lot of extensions(Including a Deno one)
+
+First, go to https://code.visualstudio.com/ and download the installer, the installation is really easy. Check out the gif below
+
+![gif](https://i.imgur.com/xvDJ2iI.gif)
+
+Once thats done open up VSC(This is how I'll refer to it from now on). It might take a while as its the first time
```

Comment:
- Space before the bracket.
- No period at the end of the sentence.
- `its` to `it is`